features/marketing-permission-and-gdpr-compatibility.md

```diff
@@ -1,2 +1,51 @@
 # Marketing permission and GDPR compatibility
+Marketing permission or newsletter constent from customers are recorded at Nosto either during order confirmation (Conversion Tracking) or through customerUpdate webhook (Customer Feed). In case of order confirmation, the consent is received through the `marketingPermission` field in `OrderCustomer` bean and though `acceptsMarketing` field in `ShopifyCustomer` bean in case of webhook. These fields tells Nosto whether the customer has given their [GDPR](https://www.eugdpr.org/) compliant consent for receiving marketing emails.
```

Comment:
Is this copy-pasted from above? I'd just have a single page with this information and link to it from other pages. Otherwise you'd need to maintain 2 separate pages.

Reply:
No. The documentation is structured in that way in this repo. Say, you are adding a documentation under feature/some_doc.md, then we create on in parent directory as Some_Doc.md. This is how it looks so far